src/header/styles.scss

```diff
@@ -220,6 +221,14 @@
   font-size: inherit;
   @include styles.info-link-spacing();
 
+  &:focus {
+    outline: none;
+  }
+
+  @include focus-visible.when-visible {
+    @include styles.link-focus;
```

Comment:
[non-blocking]: I think link-focus is confusing terminology, because it's not a link. What about focus ring?